projects/1.18/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -40,81 +44,91 @@
-  "item.gobber2.gobber2_paxel_nether": "下界多功能工具",
-  "item.gobber2.gobber2_axe_nether": "下界斧",
-  "item.gobber2.gobber2_tree_axe_nether": "下界树斧",
-  "item.gobber2.gobber2_hoe_nether": "下界锄",
-  "item.gobber2.gobber2_shovel_nether": "下界锹",
-  "item.gobber2.gobber2_bow_nether": "下界弓",
-  "item.gobber2.gobber2_sword_end": "末影剑",
+
+  "item.gobber2.gobber2_sword_nether": "下界戈伯剑",
+  "item.gobber2.gobber2_sword_nether.line1": "更好获取凋灵骷髅的头颅",
+  "item.gobber2.gobber2_pickaxe_nether": "下界戈伯镐",
+  "item.gobber2.gobber2_hammer_nether": "下界戈伯锤",
+  "item.gobber2.gobber2_excavator_nether": "下界铲",
+  "item.gobber2.gobber2_paxel_nether": "下界戈伯多功能工具",
+  "item.gobber2.gobber2_axe_nether": "下界戈伯斧",
+  "item.gobber2.gobber2_tree_axe_nether": "下界戈伯伐木斧",
+  "item.gobber2.gobber2_hoe_nether": "下界戈伯锄",
+  "item.gobber2.gobber2_shovel_nether": "下界戈伯锹",
+  "item.gobber2.gobber2_bow_nether": "下界戈伯弓",
+
+  "item.gobber2.gobber2_sword_end": "末影戈伯剑",
   "item.gobber2.gobber2_sword_sniper": "狙击手之剑",
-  "item.gobber2.gobber2_sword_sniper.line1": "一把具有特殊能力且不会损坏的剑",
-  "item.gobber2.gobber2_sword_sniper.line2": "右击以发射箭",
-  "item.gobber2.gobber2_sword_sniper.line3": "Shift+右击以切换开/关",
-  "item.gobber2.gobber2_sword_sniper.line4": "狙击能力激活：%s",
+  "item.gobber2.gobber2_sword_sniper.line1": "具有特殊能力的无法破坏之剑",
+  "item.gobber2.gobber2_sword_sniper.line2": "右键以发射箭",
+  "item.gobber2.gobber2_sword_sniper.line3": "Shift + 右键单击以切换开/关",
+  "item.gobber2.gobber2_sword_sniper.line4": "狙击能力激活: %s",
   "item.gobber2.gobber2_sword_sniper.line5": "箭数量：无限",
+
   "item.gobber2.gobber2_sword_traveler": "旅行者之剑",
-  "item.gobber2.gobber2_sword_traveler.line1": "一把具有特殊能力且不会损坏的剑",
-  "item.gobber2.gobber2_sword_traveler.line2": "右击起跳",
+  "item.gobber2.gobber2_sword_traveler.line1": "具有特殊能力的无法破坏的剑",
+  "item.gobber2.gobber2_sword_traveler.line2": "右键起跳",
   "item.gobber2.gobber2_pickaxe_end": "末影镐",
-  "item.gobber2.gobber2_hammer_end": "末影锤",
-  "item.gobber2.gobber2_excavator_end": "末影锹",
-  "item.gobber2.gobber2_paxel_end": "末影多功能工具",
+  "item.gobber2.gobber2_hammer_end": "末影戈伯锤",
+  "item.gobber2.gobber2_excavator_end": "末影戈伯铲",
+  "item.gobber2.gobber2_paxel_end": "末影戈伯多功能工具",
   "item.gobber2.gobber2_paxel_stars": "星之多功能工具",
-  "item.gobber2.gobber2_paxel_stars.line2": "右击放置火把",
-  "item.gobber2.gobber2_paxel_stars.line3": "Shift+右击以切换开/关",
-  "item.gobber2.gobber2_paxel_stars.line4": "火把功能激活：%s",
+  "item.gobber2.gobber2_paxel_stars.line1": "结合了镐，斧头和铲子，无法破坏",
+  "item.gobber2.gobber2_paxel_stars.line2": "右键放置火把",
+  "item.gobber2.gobber2_paxel_stars.line3": "Shift + 右键单击以切换开/关",
+  "item.gobber2.gobber2_paxel_stars.line4": "火把功能激活: %s",
   "item.gobber2.gobber2_paxel_stars.line5": "火把数量：无限",
-  "item.gobber2.gobber2_axe_end": "末影斧",
-  "item.gobber2.gobber2_tree_axe_end": "末影树斧",
-  "item.gobber2.gobber2_hoe_end": "末影锄",
-  "item.gobber2.gobber2_shovel_end": "末影锹",
-  "item.gobber2.gobber2_bow_end": "末影弓",
+
+  "item.gobber2.gobber2_axe_end": "末影戈伯斧",

```

```suggestion
  "item.gobber2.gobber2_axe_end": "末影斧",
```
原文`End Axe`，没有`gobber`一词。不要望文生义。